### ISSUE TITLE
The training losses in your code and in the paper.

### ISSUE THREAD

**Issue by RokiaAbdeen** [visinf/self-mono-sf#16]
Hi , 
First I'd like to thank you for this great project !
I wanna ask about the relation between the loss values (D1-all D2-all F1-all SF1-all) you have mentioned in Table 4 in the paper which reports the " Monocular scene flow evaluation on KITTI Scene Flow Training"  , and the losses that your code produces ( image reconstruction loss, point reconstruction loss, 3D smoothness loss, scene flow loss, disparity loss and total loss) . I just wanna get the same losses as you reported in your paper. could you explain how to get these loss values using your code please ?
Thanks in advance

**Comment by RokiaAbdeen** [comment]
you also have mentioned the optical flow losses (EPE and F1) and the depth losses (many values) on the testing dataset, can I get these values too on the training set? I just wanna predict how my results would be if I uploaded my results to the benchmark. or what is the relation between them?

**Comment by hurjunhwa** [comment]
Hi,

They are all included in the [loss.py](https://github.com/visinf/self-mono-sf/blob/master/losses.py).

You can find the self-supervised loss from [here](https://github.com/visinf/self-mono-sf/blob/054df08674a2df96885682e657ca4803c129a364/losses.py#L157) and the KITTI evaluation metric [here](https://github.com/visinf/self-mono-sf/blob/054df08674a2df96885682e657ca4803c129a364/losses.py#L497).

**Comment by RokiaAbdeen** [comment]
Thank you for your replay
I still wonder about these values (D1-all D2-all F1-all SF1-all) what are their corresponding losses in your code ?
and what are the corresponding losses for these values (Abs Rel Sq Rel RMSE RMSE log δ < 1:25 δ < 1:252 δ < 1:253)??

**Comment by hurjunhwa** [comment]
They are all included in the KITTI evaluation metric that I mentioned above.

Please look for the dictionary key ``d1, d2, f1, sf`` for ``D1-all D2-all F1-all SF1-all``.

It also includes monodepth evaluation metrics (Abs Rel Sq Rel RMSE etc..), using:
https://github.com/visinf/self-mono-sf/blob/054df08674a2df96885682e657ca4803c129a364/losses.py#L530

Please let me know if you have further questions!

**Comment by RokiaAbdeen** [comment]
I got it , thank you again for your replay
So, first train model with kitti-raw using train_monosf_selfsup_kitti_raw.sh script and then run the script eval_monosf_selfsup_kitti_train.sh to get the evaluation metrics values, am I right?

**Comment by hurjunhwa** [comment]
Yes.
You can also customize the code or script files for your own purposes.

**Comment by RokiaAbdeen** [comment]
I got it , 
thank u so much
